plugins/api-docs/src/components/ApiExplorerTable/ApiExplorerTable.tsx

```diff
@@ -46,16 +46,10 @@ const columns: TableColumn<Entity>[] = [
     render: (entity: any) => (
       <Link
         component={RouterLink}
-        to={generatePath(entityRoute.path, {
-          optionalNamespaceAndName: [
-            entity.metadata.namespace,
-            entity.metadata.name,
-          ]
-            .filter(Boolean)
-            .join(':'),
-          kind: entity.kind,
-          selectedTabId: 'overview',
-        })}
+        to={generatePath(
+          `/catalog/${entityRoute.path}`,
+          entityRouteParams(entity),
+        )}
```

Comment:
Maybe this is a something that we should use here too: https://github.com/backstage/backstage/blob/master/plugins/search/src/apis.ts#L50-L52

I would even prefer to make that a that a method, like `generatePathToEntity(entity)´

Reply:
The way to resolve routes into other plugins will be resolved as part of the composability work. So hopefully this will be a lot nicer soon.